### ISSUE TITLE
Usage of load-error-handling and load-media-error-handling fails because they are considered flags

### ISSUE THREAD

**Issue by michelts** [incuna/django-wkhtmltopdf#164]
These options are considered flags, as we can inspect here:

https://github.com/incuna/django-wkhtmltopdf/blob/master/wkhtmltopdf/utils.py#L40

But wkhtmltopdf expect them to have options, as we can see in the documentation:

```
      --load-error-handling <handler> Specify how to handle pages that fail to
                                      load: abort, ignore or skip (default
                                      abort)
      --load-media-error-handling <handler> Specify how to handle media files
                                      that fail to load: abort, ignore or skip
                                      (default ignore)
```

For now, I just downgraded to wkhtmltopdf 3.1.0, but I will fix it this week and check closely all other flags ignored by the commit f62662e6, to make sure we don't have any other like these :)

**Comment by pztrick** [comment]
I also had to revert to version `3.1.0`. It looks like the open P/R #161 fixes this and should be merged in.

**Comment by maxpeterson** [comment]
Fixed by #161
